python/hsfs/core/feature_logging.py

```diff
@@ -0,0 +1,51 @@
+import json
+from typing import Any, Dict
+
+import hsfs.feature_group as feature_group
+import humps
+from hsfs import util
+
+
+class FeatureLogging:
+
+    def __init__(self, id, transformed_features, untransformed_features):
+        self._id = id
+        self._transformed_features = transformed_features
+        self._untransformed_features = untransformed_features
```

Comment:
It took me a while to realize that these are actually Feature Group objects.
Can we rename them to something like `transformed_feature_group` | `untransformed_feature_group` or `transformed_features_fg` | `untransformed_features_fg` ?

Reply:
added the type hints instead to make the variable name more concise.